packages/ingest/granule.js

```diff
@@ -690,11 +650,115 @@ function copyGranuleFile(source, target, options) {
 * @param {string} target.Key - target
 * @param {Object} options - optional object with properties as defined by AWS API:
 * https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#copyObject-prop
-* @returns {Promise} returms a promise that is resolved when the file is moved
+* @returns {Promise} returns a promise that is resolved when the file is moved
 **/
 async function moveGranuleFile(source, target, options) {
   await copyGranuleFile(source, target, options);
-  return aws.s3().deleteObject(source).promise();
+  return aws.deleteS3Object(source.Bucket, source.Key);
+}
+
+/**
+* Move granule file from one s3 bucket & keypath to another
```

Comment:
This function is for moveGranuleFile`WithVersioning`, update function description.

Reply:
Good catch!

Reply:
c411cad 👍